### PR TITLE
E8M0 scale for E2M1 weights.

### PR DESCRIPTION
**Pull request by andreyanufr**
### Changes

Changed default scale fot E2M1 weights from fp16 to fp8_e8m0

### Reason for changes

According to the "OCP Microscaling Formats (MX) Specification"

### Related tickets

CVS-140944

### Tests

Will extend E2M1 tests.

### CODE REVIEW

File: nncf/tensor/functions/numeric.py [hunk 1]
```diff
@@ -858,3 +858,27 @@ def from_numpy(ndarray: np.ndarray, *, backend: TensorBackend) -> Tensor:
...
+
+    :param a: The input tensor.
+    :return: A tensor containing the base-2 logarithm of each element in a.
+    """
+    return Tensor(log2(a.data))
+
+
+@functools.singledispatch
+@tensor_guard
+def ceil(a: Tensor) -> Tensor:
```
Comment (ljaljushkin) [inline nncf/tensor/functions/numeric.py:877]: please add tests for log2 and ceil `tests/shared/test_templates/template_test_nncf_tensor.py`

**Review by ljaljushkin — CHANGES_REQUESTED** [review]

File: nncf/tensor/functions/numeric.py (hunk 1 above)
Comment (andreyanufr) [reply nncf/tensor/functions/numeric.py:877]: Done

**Review by ljaljushkin — APPROVED** [review]

**Review by alexsu52 — COMMENTED** [review]
LGTM, please, provide post_training_weight_compression for this PR.

**Comment by andreyanufr** [comment]
> LGTM, please, provide post_training_weight_compression for this PR.

80

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**